app/assets/stylesheets/alchemy/tables.scss

```diff
@@ -29,21 +29,6 @@ table {
   }
 }
 
-.floatThead-container {
```

Comment:
Do we use this class anywhere in our HTML?

Reply:
No. The plugin that gets removed with this commit adds this DOM node.